graphqlapi/dynamic_generation_meta_fetch.go

```diff
@@ -0,0 +1,650 @@
+	}
+	localMetaFetchClasses := graphql.ObjectConfig{
+		Name:        name,
+		Fields:      classFields,
+		Description: description,
+	}
+
+	return graphql.NewObject(localMetaFetchClasses), nil
+}
+
+func genMetaSingleClassField(class *models.SemanticSchemaClass) (*graphql.Field, error) {
+
+	metaClassName := mergeStrings("Meta", class.Class)
+
+	singleClassPropertyFields := graphql.ObjectConfig{
+		Name: metaClassName,
+		Fields: (graphql.FieldsThunk)(func() graphql.Fields {
+			singleClassPropertyFields, err := genMetaSingleClassPropertyFields(class)
+			if err != nil {
+				panic("oops")
+			}
+			return singleClassPropertyFields
+		}),
+		Description: "Type of fetch on the internal Weaviate",
+	}
+
+	singleClassPropertyFieldsObject := graphql.NewObject(singleClassPropertyFields)
+	singleClassPropertyFieldsField := &graphql.Field{
+		Type:        singleClassPropertyFieldsObject,
+		Description: class.Description,
+		Args: graphql.FieldConfigArgument{
+			"_certainty": &graphql.ArgumentConfig{
+				Description: "How certain about these values?",
+				Type:        graphql.Float,
+			},
+			"_limit": &graphql.ArgumentConfig{
+				Description: "define the max returned values",
+				Type:        graphql.Int,
+			},
+			"_skip": &graphql.ArgumentConfig{
+				Description: "define the amount of values to skip",
+				Type:        graphql.Int,
+			},
+		},
+		Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+			return nil, fmt.Errorf("Not supported")
+		},
+	}
+	return singleClassPropertyFieldsField, nil
+}
+
+func genMetaSingleClassPropertyFields(class *models.SemanticSchemaClass) (graphql.Fields, error) {
+
+	singleClassPropertyFields := graphql.Fields{}
+
+	metaPropertyObj, err := genMetaPropertyObj(class)
+	if err != nil {
+		return nil, fmt.Errorf("Could not build GraphQL schema, because: %v", err)
+	}
+	metaPropertyObjField := &graphql.Field{
+		Description: "meta information about class object",
+		Type:        metaPropertyObj,
+		Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+			return nil, fmt.Errorf("Not supported")
+		},
+	}
+	singleClassPropertyFields["Meta"] = metaPropertyObjField
+
+	for _, property := range class.Properties {
+
+		propertyType, err := schema.GetPropertyDataType(class, property.Name)
+		if err != nil {
+			return nil, err
+		}
+
+		convertedDataType, err := handleMetaFetchNonObjectDataTypes(*propertyType, class, property)
+
+		if err != nil {
+			return nil, err
+		}
+		singleClassPropertyFields[property.Name] = convertedDataType
+	}
+	return singleClassPropertyFields, nil
+}
+
+func handleMetaFetchNonObjectDataTypes(dataType schema.DataType, class *models.SemanticSchemaClass, property *models.SemanticSchemaClassProperty) (*graphql.Field, error) {
+
+	metaClassStringPropertyFields, err := genMetaClassStringPropertyFields(class, property)
+	if err != nil {
+		return nil, err
+	}
+	metaClassIntPropertyFields, err := genMetaClassIntPropertyFields(class, property)
```

Comment:
You probably want a newline before this line (same for the other cases in this function).